spanner/spanner_samples.rb

```diff
@@ -968,6 +968,38 @@ def delete_using_partitioned_dml project_id:, instance_id:, database_id:
   # [END spanner_dml_partitioned_delete]
 end
 
+def update_using_batch_dml project_id:, instance_id:, database_id:
+  # [START spanner_dml_batch_update]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+  insert_statement = "INSERT INTO Albums "\
```

Comment:
What about moving insert and update statements inside the transaction not using variables?

Reply:
Done.